src/components/blog/category copy.js

```diff
@@ -0,0 +1,179 @@
+import React, { useState, useEffect } from "react";
+ 
+import axios from 'axios'
+import Swal from 'sweetalert2';
+import { useNavigate } from 'react-router-dom'
+import { isAuthenticated } from '../../authService';
+import { Sidebar, Menu, MenuItem, SubMenu } from 'react-pro-sidebar';
+import  {CustomSidebar} from '../utilities/sidebar.js'
+import Select from 'react-select';
+import DataTable from 'react-data-table-component';
+import ReactPaginate from 'react-paginate';
+import CategoryList from './category_list.js'
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'; 
+import {  faPlus} from '@fortawesome/free-solid-svg-icons';
+
+export default function CreateCategory() {
+ 
+  const navigate = useNavigate();
+  const [category_name, setcategory_name] = useState("")
```

Comment:
variable naming should be consistent throughout the code, I can see multiple conventions followed here.

snake_case (category_name)
normalcase (newcatadd)
PascalCase (EditID)
camelCase (selectedOption)

In react generally camelCase is followed and for classes and components  PascalCase is followed

Reply:
Make sure through out the project variable naming is consistent.